util/topgen/templates/xbar_env_pkg__params.sv.tpl

```diff
@@ -51,12 +51,16 @@
 // List of Xbar device memory map
 tl_device_t xbar_devices[$] = '{
 % for xbar in top["xbar"]:
+<%
+  # TODO: Add support for multiple ASIDs within an xbar.
```

Comment:
Is this TODO really needed or just speculative? If the latter, please remove it.

Reply:
An issue should still be created for the missing feature (and set to an appropriate priority level, even if it's P4 "probably never"), and the limitations should be documented. It needs to be obvious that the support for multiple address spaces is _only_ for entirely disjoint networks. With the current code, two hosts using different address spaces can't connect to the same device node.

Due to this limitation, IPs need to have a separate node defined for each address space, even if the hosts would actually access the same CSRs with the same privileges (tail-wagging-dog property flow). For the current set of known use cases, this seems to be fine, however.

Reply:
I created https://github.com/lowRISC/opentitan/issues/24801 to track that.